lib/profiles/WD-Echidna.js

```diff
@@ -0,0 +1,23 @@
+
+// Working Draft profile
+
+exports.name = "WD-Echidna";
+exports.config = {
+    status:             "WD"
+,   longStatus:         "Working Draft"
+,   previousVersion:    true
+,   styleSheet:         "W3C-WD"
+,   stabilityWarning:   true
+,   recTrackStatus:     true
+};
+
```

Comment:
You probably don't need to copy the configuration, either. You can just copy the `config` exported by `WD.js`.